frontend/src/pages/orgs/[orgId]/events/[eventId].tsx

```diff
@@ -126,141 +131,244 @@ const EventAdminPage: NextPage = () => {
           {openEditEvent && (
             <EditEvent open={openEditEvent} onClose={() => setOpenEditEvent(false)} event={data.event} />
           )}
-          <Grid container direction="column" spacing={5}>
-            <Grid item>
-              <Typography variant="h1" align="center">
-                {data.event.title}
-              </Typography>
+          <Box mb={10}>
+            {openEditEvent && (
+              <EditEvent open={openEditEvent} onClose={() => setOpenEditEvent(false)} event={data.event} />
+            )}
+            <Grid>
+              <Grid container direction="column" spacing={5}>
+                <Grid item>
+                  <Typography variant="h2" align="center">
+                    {data.event.title}
+                  </Typography>
+                </Grid>
+              </Grid>
             </Grid>
-            <Grid item container spacing={5}>
-              <Grid item xs={4}>
-                <Card variant="outlined">
-                  <CardHeader title="Generell informasjon" />
-                  <CardActions>
-                    <Button startIcon={<Edit />} onClick={() => setOpenEditEvent(true)}>
-                      Rediger
-                    </Button>
-                  </CardActions>
-                  <CardContent>
-                    <List>
-                      {stringEventFields.map((headerPair: HeaderValuePair<Event>) =>
-                        renderInfo(headerPair.header, data.event[headerPair.field] as string)
-                      )}
-                      {dateEventFields.map((headerPair: HeaderValuePair<Event>) =>
-                        renderInfo(
-                          headerPair.header,
-                          data.event[headerPair.field]
-                            ? dayjs(data.event[headerPair.field] as string).format("HH:mm DD-MM-YYYY")
-                            : ""
-                        )
+          </Box>
+          <Accordion elevation={0} defaultExpanded={true} classes={{ root: classes.accordion }}>
+            <Box py={2}>
+              <AccordionSummary aria-controls="panel1a-content" id="panel1a-header">
+                <Grid container alignItems="center" justifyContent="space-between">
+                  <Grid item xs={12} sm="auto">
+                    <Box ml={5}>
+                      <CardHeader titleTypographyProps={{ variant: "h4" }} title="Generell informasjon" />
+                    </Box>
```

Comment:
Unless we're in a `<Card />`-component, it is better to just utilize the `<Typography />`-component directly.